.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   },
   ignorePatterns: ['.eslintrc.js'],
   rules: {
+    '@typescript-eslint/no-floating-promises': 1,
```

Comment:
it is irrelevant but I am curious. Should we use `on` here? Is this even available - I have no clue of eslint config. But seeing `off` being used below. I thought this would make it consistent.